eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -19,344 +19,336 @@ parameters:
-        echo "find ./sdk \( -iname '*.hpp' -o -iname '*.cpp' \) ! -iname 'json.hpp' -exec clang-format -i {} \;"
-        echo ""
-        echo "List of files not formatted correctly:"
-        git status | grep modified | awk '{print $2}'
-        exit 1
-      fi
-
-      echo Success, all files are formatted correctly according to the .clang-format file.
-      exit 0
-
-    displayName: Validate Clang Format
-    condition: and(succeededOrFailed(), eq(variables['CHECK_CLANG_FORMAT'], 1))
-  
-  - ${{ each artifact in parameters.Artifacts }}: 
-    - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
-      parameters:
-        PackageName: ${{ artifact.Name }}
-        ServiceDirectory: ${{ parameters.ServiceDirectory }}
-        TestPipeline: ${{ parameters.TestPipeline }}
-
-  # Install apt dependencies (if appropriate)
-  - bash: sudo apt install -y $(AptDependencies)
-    condition: and(succeededOrFailed(), ne(variables['AptDependencies'], ''))
-    displayName: Install dependencies from apt
-
-  - template: /eng/pipelines/templates/steps/vcpkg.yml
-    parameters:
-      DependenciesVariableName: VcpkgInstall
-
-  - script: |
-      dotnet tool install -g dotnet-reportgenerator-globaltool
-      dotnet tool install dotnet-reportgenerator-globaltool --tool-path tools
-    displayName: Install coverage tools
-    # CODE_COVERAGE variable is '' (do-not-generate) in all matrix but linux-gcc
-    # It is 'enabled' by default on linux-gcc but it can be opt-out by each pipeline (disabled) 
-    condition: and(succeededOrFailed(), ne(variables['CODE_COVERAGE'], 'disabled'), ne(variables['CODE_COVERAGE'], ''))
-
-  - template: /eng/pipelines/templates/steps/cmake-build.yml
-    parameters:
-      GenerateArgs: "$(CmakeArgs)"
-      BuildArgs: "$(BuildArgs)"
-
-  - script: ctest -C Debug -V --tests-regex ${{ parameters.CtestRegex }}
-    workingDirectory: build
-    displayName: ctest
-
-  # Make coverage targets (specified in coverage_targets.txt) and assemble
-  # coverage report
-  - bash: |
-      make `cat ${{ parameters.ServiceDirectory }}-targets-coverage.txt`
-      ../tools/reportgenerator "-reports:${{ parameters.CoverageReportPath }}" "-targetdir:." "-reporttypes:Cobertura"
-    workingDirectory: build
-    displayName: Generate Code Coverage Data
-    condition: and(succeededOrFailed(), ne(variables['CODE_COVERAGE'], 'disabled'), ne(variables['CODE_COVERAGE'], ''))
-  
-  - task: PublishCodeCoverageResults@1
-    inputs:
-      codeCoverageTool: Cobertura
-      summaryFileLocation: '$(Build.SourcesDirectory)/**/Cobertura.xml'
-    displayName: Publish Code Coverage to DevOps
-    condition: and(succeededOrFailed(), ne(variables['CODE_COVERAGE'], 'disabled'), ne(variables['CODE_COVERAGE'], ''))
-
-# Disable build for cpp - client
-- ${{ if ne(parameters.ServiceDirectory, 'not-specified' )}}:
-  - job: GenerateReleaseArtifacts
+  - job: Validate
```

Comment:
What caused all these new whitespace changes to this file? Will someone else editing this cause another set of whitespace changes?